src/Microsoft.Diagnostics.Monitoring/Microsoft.Diagnostics.Monitoring.csproj

```diff
@@ -17,7 +17,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="$(MicrosoftBclAsyncInterfacesVersion)" />
+    <PackageReference Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'" Include="Microsoft.Bcl.AsyncInterfaces" Version="$(MicrosoftBclAsyncInterfacesVersion)" />
```

Comment:
This package currently targets only .NET Standard 2.0; I left it in case we multi-target, to not forget it.